x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/components/step_define/step_define_form.tsx

```diff
@@ -35,6 +35,8 @@ import { AggListForm } from '../aggregation_list';
 import { GroupByListForm } from '../group_by_list';
 import { SourceIndexPreview } from '../source_index_preview';
 import { PivotPreview } from './pivot_preview';
+// @ts-ignore: could not find declaration file for module
+import { KqlFilterBar } from '../../../../../components/kql_filter_bar';
```

Comment:
is it worth adding a definition file for this rather than suppressing the error. Looking at the class it'd be something like:
```
import { FC } from 'react';
declare const KqlFilterBar: FC<{
  indexPattern: any;
  initialValue?: string;
  onSubmit: (kqlQueryValues: any) => void;
  placeholder?: string;
  valueExternal?: string;
}>;

```
`indexPattern` and `onSubmit` could have better types if it's easy to do.